test/unit/spec/narrative_core/upload/importSetup-spec.js

```diff
@@ -282,7 +290,27 @@ define([
             });
 
             // TODO
-            xit('should error properly when unable to find bulk specification info', async () => {});
+            it('should error properly when unable to find bulk specification info', async () => {
+                // see https://github.com/kbase/staging_service/tree/develop#error-response-12
+                // for error details
+                const filename = 'xsv_input.csv';
+                stubBulkSpecificationRequest(404, 'not found', {
+                    errors: [
+                        {
+                            type: 'cannot_find_file',
+                            file: filename,
+                        },
+                    ],
```

Comment:
should this be stringified like the other stubbed request?

Reply:
Nope, but the next one is! There's another case that should be tested in it that isn't fleshed out yet - will be in the next PR.

Reply:
Yes. Done!